target/linux/bcm63xx/patches-5.4/806-board_bcm6328-technicolor-tg582n.patch

```diff
@@ -0,0 +1,70 @@
+Index: linux-5.4.61/arch/mips/bcm63xx/boards/board_bcm963xx.c
+===================================================================
+--- linux-5.4.61.orig/arch/mips/bcm63xx/boards/board_bcm963xx.c
++++ linux-5.4.61/arch/mips/bcm63xx/boards/board_bcm963xx.c
```

Comment:
Please, refresh the patch (make target/linux/refresh).
Also, this should now be integrated into https://github.com/openwrt/openwrt/blob/master/target/linux/bcm63xx/patches-5.4/512-board_bcm6328.patch